testsuite/forge-cli/src/main.rs

```diff
@@ -1950,9 +1950,9 @@ fn realistic_env_max_load_test(
         .add_system_metrics_threshold(SystemMetricsThreshold::new(
             // Check that we don't use more than 18 CPU cores for 10% of the time.
             MetricsThreshold::new(18.0, 10),
-            // Memory starts around 3GB, and grows around 1.2GB/hr in this test.
+            // Memory starts around 3.5GB, and grows around 1.4GB/hr in this test.
```

Comment:
I put here what values I saw from the chart, and then added the buffer below into the requirements, so these were intentionally different.

maybe add a comment that we are adding a buffer, and leave the values as they really are in the comment?

Reply:
Here's the latest chart after these changes (that also increases TPS). Do the values make sense for this chart? https://aptoslabs.grafana.net/d/Rgf4wanVk/node-resource-metrics?from=1717664422000&to=1717665130000&var-Datasource=fHo-R604z&var-BigQuery=axNEitxVz&var-metrics_source=All&var-chain_name=forge-1&var-cluster=All&var-namespace=forge-e2e-pr-13558&var-kubernetes_pod_name=All&orgId=1&viewPanel=2

Reply:
hm, you might need even higher starting value - like 3.7 or 4

for the rate, we need 2hr run to see - so we can land and iterate.